### PR TITLE
EVG-16501: clean up error message style

### PR DESCRIPTION
**Pull request by Kimchelly**
Jira: https://jira.mongodb.org/browse/EVG-16501

* Clean up error message style.
* Update usages of grip catcher.

### CODE REVIEW

File: greenbay/check/compile_golang.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func (c compileGolang) Compile(testBody string, _ ...string) error {
 
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		return errors.Wrapf(err, "problem compiling go test: %s", string(out))
+		return errors.Wrapf(err, "compiling go test: %s", string(out))
```
Comment (julianedwards) [inline greenbay/check/compile_golang.go:100]: `Go`

File: greenbay/check/compile_windows.go [hunk 2]
```diff
@@ -55,14 +55,14 @@ func newCompileVS() compiler {
 	k, err := registry.OpenKey(registry.LOCAL_MACHINE, vcKeyName,
 		registry.ENUMERATE_SUB_KEYS|registry.READ)
 	if err != nil {
-		c.catcher.Add(errors.Wrap(err, "problem reading from registry"))
+		c.catcher.Add(errors.Wrap(err, "reading from registry"))
```
Comment (julianedwards) [inline greenbay/check/compile_windows.go:58]: `c.catcher.Wrap`

Comment (julianedwards) [inline greenbay/check/compile_windows.go:58]: `cather.Wrap`

File: greenbay/check/compile_windows.go [hunk 3]
```diff
@@ -55,14 +55,14 @@ func newCompileVS() compiler {
...
 		return nil
 	}
 	defer k.Close()
 
 	installed, err := k.ReadSubKeyNames(-1)
 	if err != nil {
-		c.catcher.Add(errors.Wrap(err, "problem reading subkeys"))
+		c.catcher.Add(errors.Wrap(err, "reading subkeys"))
```
Comment (julianedwards) [inline greenbay/check/compile_windows.go:65]: `c.catcher.Wrap`

Comment (julianedwards) [inline greenbay/check/compile_windows.go:65]: `c.catcher.Wrap`

File: greenbay/client.go [hunk 4]
```diff
@@ -28,17 +28,17 @@ type Client struct {
...
 	conf, err := ReadConfig(confPath)
 	if err != nil {
-		return nil, errors.Wrap(err, "problem parsing config file")
+		return nil, errors.Wrap(err, "parsing config file")
 	}
 
 	c, err := rest.NewQueueClient(host, port, "")
 	if err != nil {
-		return nil, errors.Wrap(err, "problem constructing amboy rest client")
+		return nil, errors.Wrap(err, "constructing amboy rest client")
```
Comment (julianedwards) [inline greenbay/client.go:41]: `Amboy`

Comment (julianedwards) [inline greenbay/client.go:41]: `Amboy`

File: greenbay/config_parsing.go [hunk 5]
```diff
@@ -51,12 +51,12 @@ func getJSONFormattedConfig(format format, data []byte) ([]byte, error) {
 func getRawConfig(fn string) ([]byte, error) {
 	data, err := ioutil.ReadFile(fn)
 	if err != nil {
-		return nil, errors.Wrapf(err, "problem reading greenbay config file: %s", fn)
+		return nil, errors.Wrapf(err, "reading greenbay config file '%s'", fn)
```
Comment (julianedwards) [inline greenbay/config_parsing.go:54]: `Greenbay`

File: greenbay/config_raw.go [hunk 6]
```diff
@@ -17,11 +17,11 @@ type rawTest struct {
 func (t *rawTest) resolveCheck() (Checker, error) {
 	check, err := t.getChecker()
 	if err != nil {
-		return nil, errors.Wrap(err, "problem determining job type")
+		return nil, errors.Wrap(err, "determining job type")
 	}
 
 	if err = json.Unmarshal(t.RawArgs, check); err != nil {
-		return nil, errors.Wrapf(err, "problem parsing argument for job %s (%s)",
+		return nil, errors.Wrapf(err, "parsing argument for job %s (%s)",
```
Comment (julianedwards) [inline greenbay/config_raw.go:24]: `'%s'`

Comment (julianedwards) [inline greenbay/config_raw.go:24]: `'%s'`

File: greenbay/output_gotest.go [hunk 7]
```diff
@@ -41,7 +41,7 @@ func (r *GoTest) Populate(jobs <-chan amboy.Job) error {
 // ToFile writes the "go test -v" output to a file.
 func (r *GoTest) ToFile(fn string) error {
 	if err := ioutil.WriteFile(fn, r.buf.Bytes(), 0644); err != nil {
-		return errors.Wrapf(err, "problem writing output to %s", fn)
+		return errors.Wrapf(err, "writing output to %s", fn)
```
Comment (julianedwards) [inline greenbay/output_gotest.go:44]: `'%s'`

Comment (julianedwards) [inline greenbay/output_gotest.go:44]: `'%s'`

File: greenbay/output_json_compser.go [hunk 8]
```diff
@@ -17,7 +17,7 @@ func (o *jsonOutput) Loggable() bool   { return true }
 func (o *jsonOutput) String() string {
 	out, err := json.Marshal(o)
 	if err != nil {
-		return fmt.Sprintf("error processing result for %s (%+v): %+v",
+		return fmt.Sprintf("processing result for %s (%+v): %+v",
```
Comment (julianedwards) [inline greenbay/output_json_compser.go:20]: `'%s'`

Comment (julianedwards) [inline greenbay/output_json_compser.go:20]: `'%s'`

File: greenbay/service.go [hunk 9]
```diff
@@ -49,11 +49,11 @@ func NewService(confPath string, host string, port int) (*Service, error) {
 	app := s.service.App()
 
 	if err := app.SetPort(port); err != nil {
-		return nil, errors.Wrap(err, "problem constructing greenbay service")
+		return nil, errors.Wrap(err, "constructing greenbay service")
```
Comment (julianedwards) [inline greenbay/service.go:52]: `Greenbay`

File: greenbay/service.go [hunk 10]
```diff
@@ -49,11 +49,11 @@ func NewService(confPath string, host string, port int) (*Service, error) {
...
 	}
 
 	if err := app.SetHost(host); err != nil {
-		return nil, errors.Wrap(err, "problem constructing greenbay service")
+		return nil, errors.Wrap(err, "constructing greenbay service")
```
Comment (julianedwards) [inline greenbay/service.go:56]: `Greenbay`

File: operations/ftdc.go [hunk 11]
```diff
@@ -369,20 +369,20 @@ func toCSV() cli.Command {
...
+					return errors.Wrapf(err, "opening file '%s'", outputPath)
 				}
 				defer func() { grip.EmergencyFatal(outputFile.Close()) }()
 			}
 
 			// actually convert data
 			//
 			if err := ftdc.WriteCSV(ctx, ftdc.ReadChunks(ctx, inputFile), outputFile); err != nil {
-				return errors.Wrap(err, "problem parsing csv")
+				return errors.Wrap(err, "parsing csv")
```
Comment (julianedwards) [inline operations/ftdc.go:385]: `CSV`

Comment (julianedwards) [inline operations/ftdc.go:385]: `CSV`

File: operations/ftdc.go [hunk 12]
```diff
@@ -562,7 +562,7 @@ func toMDB() cli.Command {
 					break
 				}
 			}
-			catcher.Add(errors.Wrapf(iter.Err(), "problem iterating ftdc data from file '%s'", srcPath))
+			catcher.Add(errors.Wrapf(iter.Err(), "iterating FTDC data from file '%s'", srcPath))
```
Comment (julianedwards) [inline operations/ftdc.go:565]: `catcher.Wrapf`

Comment (julianedwards) [inline operations/ftdc.go:565]: `catcher.Wrapf`

File: operations/notify.go [hunk 13]
```diff
@@ -115,7 +115,7 @@ func Notify() cli.Command {
 				}
 				sender, err = send.MakeSlackLogger(opts)
 				if err != nil {
-					return errors.Wrap(err, "problem building slack logger")
+					return errors.Wrap(err, "building slack logger")
```
Comment (julianedwards) [inline operations/notify.go:118]: `Slack`

Comment (julianedwards) [inline operations/notify.go:118]: `Slack`

File: operations/notify.go [hunk 14]
```diff
@@ -129,18 +129,18 @@ func Notify() cli.Command {
...
 				if err != nil {
-					return errors.Wrap(err, "problem building email logger")
+					return errors.Wrap(err, "building email logger")
 				}
 			case "xmpp":
 				sender, err = send.MakeXMPP(c.String("target"))
 
 				if err != nil {
-					return errors.Wrap(err, "problem building jabber/xmpp logger")
+					return errors.Wrap(err, "building jabber/xmpp logger")
```
Comment (julianedwards) [inline operations/notify.go:143]: `Jabber/XMPP`?

Comment (julianedwards) [inline operations/notify.go:143]: `Jaber/XMPP`?

File: operations/repo.go [hunk 15]
```diff
@@ -188,18 +188,18 @@ func submitRepo(ctx context.Context, opts submitRepoOptions) error {
 
 	client, err := barquesubmit.New(opts.url)
 	if err != nil {
-		return errors.Wrap(err, "problem constructing barque client")
+		return errors.Wrap(err, "constructing barque client")
```
Comment (julianedwards) [inline operations/repo.go:191]: `Barque`

Comment (julianedwards) [inline operations/repo.go:191]: `Barque`

File: operations/repo.go [hunk 16]
```diff
@@ -188,18 +188,18 @@ func submitRepo(ctx context.Context, opts submitRepoOptions) error {
...
 	}
 
 	if opts.username != "" && opts.apiKey != "" {
 		client.SetCredentials(opts.username, opts.apiKey)
 	} else if err = client.Login(ctx, opts.username, opts.password); err != nil {
-		return errors.Wrap(err, "problem authenticating to barque")
+		return errors.Wrap(err, "authenticating to barque")
```
Comment (julianedwards) [inline operations/repo.go:197]: `Barque`

Comment (julianedwards) [inline operations/repo.go:197]: `Barque`

File: operations/sthree.go [hunk 17]
```diff
@@ -112,12 +112,12 @@ func s3PutCmd() cli.Command {
...
 			bucket, err := pail.NewS3Bucket(opts)
 			if err != nil {
-				return errors.Wrap(err, "problem getting new bucket")
+				return errors.Wrap(err, "getting new bucket")
 			}
 
 			fmt.Printf("Uploading file '%s' to path '%s' in bucket '%s'\n", c.String("file"), objectKey, bucketName)
 			if err := bucket.Upload(ctx, objectKey, c.String("file")); err != nil {
-				return errors.Wrapf(err, "problem putting %s in s3", c.String("file"))
+				return errors.Wrapf(err, "putting file '%s' in s3", c.String("file"))
```
Comment (julianedwards) [inline operations/sthree.go:120]: `S3`

Comment (julianedwards) [inline operations/sthree.go:120]: `S3`

File: operations/sthree.go [hunk 18]
```diff
@@ -157,12 +157,12 @@ func s3GetCmd() cli.Command {
...
 			bucket, err := pail.NewS3Bucket(opts)
 			if err != nil {
-				return errors.Wrap(err, "problem getting new bucket")
+				return errors.Wrap(err, "getting new bucket")
 			}
 
 			return errors.Wrapf(
 				bucket.Download(ctx, c.String("name"), c.String("file")),
-				"problem getting %s from s3",
+				"getting object '%s' from s3",
```
Comment (julianedwards) [inline operations/sthree.go:165]: `S3`

Comment (julianedwards) [inline operations/sthree.go:165]: `S3`

File: operations/sthree.go [hunk 19]
```diff
@@ -196,12 +196,12 @@ func s3DeleteCmd() cli.Command {
...
 			bucket, err := pail.NewS3Bucket(opts)
 			if err != nil {
-				return errors.Wrap(err, "problem getting new bucket")
+				return errors.Wrap(err, "getting new bucket")
 			}
 
 			return errors.Wrapf(
 				bucket.Remove(ctx, c.String("name")),
-				"problem removing %s from s3",
+				"removing '%s' from s3",
```
Comment (julianedwards) [inline operations/sthree.go:204]: `S3`

Comment (julianedwards) [inline operations/sthree.go:204]: `S3`

File: operations/sthree.go [hunk 20]
```diff
@@ -235,12 +235,12 @@ func s3DeletePrefixCmd() cli.Command {
...
 			bucket, err := pail.NewS3Bucket(opts)
 			if err != nil {
-				return errors.Wrap(err, "problem getting new bucket")
+				return errors.Wrap(err, "getting new bucket")
 			}
 
 			return errors.Wrapf(
 				bucket.RemovePrefix(ctx, c.String("name")),
-				"problem removing %s from s3",
+				"removing prefix '%s' from s3",
```
Comment (julianedwards) [inline operations/sthree.go:243]: `S3`

Comment (julianedwards) [inline operations/sthree.go:243]: `S3`

File: operations/sthree.go [hunk 21]
```diff
@@ -274,12 +274,12 @@ func s3DeleteMatchingCmd() cli.Command {
...
 			bucket, err := pail.NewS3Bucket(opts)
 			if err != nil {
-				return errors.Wrap(err, "problem getting new bucket")
+				return errors.Wrap(err, "getting new bucket")
 			}
 
 			return errors.Wrapf(
 				bucket.RemoveMatching(ctx, c.String("match")),
-				"problem removing objects matching %s in s3",
+				"removing objects matching pattern '%s' in s3",
```
Comment (julianedwards) [inline operations/sthree.go:282]: `S3`

Comment (julianedwards) [inline operations/sthree.go:282]: `S3`

File: operations/sthree.go [hunk 22]
```diff
@@ -335,7 +335,7 @@ func s3SyncToCmd() cli.Command {
 			}
 			return errors.Wrapf(
 				bucket.Push(ctx, syncOpts),
-				"problem syncing %s to s3",
+				"syncing %s to s3",
```
Comment (julianedwards) [inline operations/sthree.go:338]: `'%s'` and `S3`

Comment (julianedwards) [inline operations/sthree.go:338]: `'%s'` and `S3`

File: operations/sthree.go [hunk 23]
```diff
@@ -390,7 +390,7 @@ func s3SyncFromCmd() cli.Command {
 			}
 			return errors.Wrapf(
 				bucket.Pull(ctx, syncOpts),
-				"problem syncing %s from  s3",
+				"syncing %s from s3",
```
Comment (julianedwards) [inline operations/sthree.go:393]: `'%s'` and `S3`

Comment (julianedwards) [inline operations/sthree.go:393]: `'%s'` and `S3`

File: operations/sthree.go [hunk 24]
```diff
@@ -531,7 +531,7 @@ func setVerboseLogging(verbose bool) error {
 		info := sender.Level()
 		info.Threshold = level.Debug
 
-		return errors.Wrap(sender.SetLevel(info), "problem setting logger threshold to debug")
+		return errors.Wrap(sender.SetLevel(info), "setting logger threshold to debug")
```
Comment (julianedwards) [inline operations/sthree.go:534]: `to info`

File: operations/version.go [hunk 25]
```diff
@@ -53,7 +53,7 @@ func Version() cli.Command {
 			if isJSON {
 				out, err := json.MarshalIndent(info, "", "   ")
 				if err != nil {
-					return errors.Wrap(err, "problem marshaling json")
+					return errors.Wrap(err, "marshalling json")
```
Comment (julianedwards) [inline operations/version.go:56]: `JSON`

Comment (julianedwards) [inline operations/version.go:56]: `JSON`

File: repobuilder/config.go [hunk 26]
```diff
@@ -131,7 +130,7 @@ func (c *RepositoryConfig) processRepos() error {
 	for idx, dfn := range c.Repos {
 		// do some basic validation that the type value is correct.
 		if dfn.Type != DEB && dfn.Type != RPM {
-			catcher.Add(fmt.Errorf("%s is not a valid repo type", dfn.Type))
+			catcher.Errorf("%s is not a valid repo type", dfn.Type)
```
Comment (julianedwards) [inline repobuilder/config.go:133]: `'%s'`

Comment (julianedwards) [inline repobuilder/config.go:133]: `'%s'`

File: repobuilder/config.go [hunk 27]
```diff
@@ -142,14 +141,12 @@ func (c *RepositoryConfig) processRepos() error {
...
-			catcher.Add(fmt.Errorf("the %s.%s already exists as repo #%d",
-				dfn.Edition, dfn.Name, idx))
+			catcher.Errorf("'%s.%s' already exists as repo #%d", dfn.Edition, dfn.Name, idx)
 			continue
 		}
 
 		if dfn.Type == DEB && len(dfn.Architectures) == 0 {
-			catcher.Add(fmt.Errorf("debian distro %s does not specify architecture list",
-				dfn.Name))
+			catcher.Errorf("debian distro '%s' does not specify architecture list", dfn.Name)
```
Comment (julianedwards) [inline repobuilder/config.go:149]: `Debian`

Comment (julianedwards) [inline repobuilder/config.go:149]: `Debian`

**Review by julianedwards — CHANGES_REQUESTED** [review]

File: repobuilder/config.go (hunk 27 above)
Comment (Kimchelly) [reply repobuilder/config.go]: Fixed.

File: repobuilder/config.go (hunk 26 above)
Comment (Kimchelly) [reply repobuilder/config.go]: Fixed.

File: operations/version.go (hunk 25 above)
Comment (Kimchelly) [reply operations/version.go]: Fixed.

File: operations/sthree.go (hunk 24 above)
Comment (Kimchelly) [reply operations/sthree.go:534]: It's setting the debug threshold on the level info.

File: operations/sthree.go (hunk 23 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 22 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 21 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 20 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 19 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 18 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/sthree.go (hunk 17 above)
Comment (Kimchelly) [reply operations/sthree.go]: Fixed.

File: operations/repo.go (hunk 16 above)
Comment (Kimchelly) [reply operations/repo.go]: Fixed.

File: operations/repo.go (hunk 15 above)
Comment (Kimchelly) [reply operations/repo.go]: Fixed.

File: operations/notify.go (hunk 14 above)
Comment (Kimchelly) [reply operations/notify.go]: Fixed.

File: operations/notify.go (hunk 13 above)
Comment (Kimchelly) [reply operations/notify.go]: Fixed.

File: operations/ftdc.go (hunk 12 above)
Comment (Kimchelly) [reply operations/ftdc.go]: Fixed.

File: operations/ftdc.go (hunk 11 above)
Comment (Kimchelly) [reply operations/ftdc.go]: Fixed.

File: greenbay/service.go (hunk 9 above)
Comment (Kimchelly) [reply greenbay/service.go]: Fixed.

File: greenbay/service.go (hunk 10 above)
Comment (Kimchelly) [reply greenbay/service.go]: Fixed.

File: greenbay/output_json_compser.go (hunk 8 above)
Comment (Kimchelly) [reply greenbay/output_json_compser.go]: Fixed.

File: greenbay/output_gotest.go (hunk 7 above)
Comment (Kimchelly) [reply greenbay/output_gotest.go]: Fixed.

File: greenbay/config_raw.go (hunk 6 above)
Comment (Kimchelly) [reply greenbay/config_raw.go]: Fixed.

File: greenbay/config_parsing.go (hunk 5 above)
Comment (Kimchelly) [reply greenbay/config_parsing.go]: Fixed.

File: greenbay/client.go (hunk 4 above)
Comment (Kimchelly) [reply greenbay/client.go]: Fixed.

File: greenbay/check/compile_windows.go (hunk 3 above)
Comment (Kimchelly) [reply greenbay/check/compile_windows.go]: Fixed.

File: greenbay/check/compile_windows.go (hunk 2 above)
Comment (Kimchelly) [reply greenbay/check/compile_windows.go]: Fixed.

File: greenbay/check/compile_golang.go (hunk 1 above)
Comment (Kimchelly) [reply greenbay/check/compile_golang.go]: Fixed.

**Review by Kimchelly — COMMENTED** [review]
There's a bunch of duplicate comments but I think I got all of them.

**Review by julianedwards — APPROVED** [review]

**Pull request merged**